client-side/src/index.js

```diff
@@ -1,16 +1,33 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import { BrowserRouter } from 'react-router-dom';
 import reportWebVitals from './reportWebVitals';
-import App from './App';
+import { ApolloProvider, ApolloClient, InMemoryCache } from '@apollo/client';
+import { Provider } from 'react-redux';
+import { store } from './redux/store.jsx';
+import App from './App.jsx';
 import './index.scss';
 
 
+const port = process.env.REACT_APP_PORT
+const url = process.env.REACT_APP_URL
+
+const client = new ApolloClient({
+
+  uri: `${url}:${port}`,
+
+  cache: new InMemoryCache()
+
+});
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
+
   <React.StrictMode>
-    <App/>
+    <ApolloProvider client={client}>
```

Comment:
does apolloProvider needs to be declare here? Can't it be declared in statistics page only?